src/include/txn.i

```diff
@@ -750,27 +750,34 @@ __wt_txn_read(WT_SESSION_IMPL *session, WT_UPDATE *upd, WT_UPDATE **updp)
 {
     static WT_UPDATE tombstone = {.txnid = WT_TXN_NONE, .type = WT_UPDATE_TOMBSTONE};
     WT_VISIBLE_TYPE upd_visible;
+    uint8_t type;
     bool skipped_birthmark;
 
     *updp = NULL;
+
+    type = 0; /* [-Wconditional-uninitialized] */
```

Comment:
I suggest WT_UPDATE_INVALID (and remove the comment).

Reply:
Good point, I pushed the `WT_UPDATE_INVALID` change in 19aae8e23.

I don't see the point of removing the comment, unless there's a real path where the value can be used uninitialized?